### ISSUE TITLE
Video quality is degrading, when meeting recording via Jibri starting

### ISSUE THREAD

**Issue by SignFinder** [jitsi/jitsi-meet#14019]
We connected Jibri to our Jitsi infrastructure and found that when recording is starting - quality of video of the peoples video stream on the meeting screen is degrading to 360.

To test it we deployed test lab with stable Jitsi+Jibri. It contains three machines for Jitsi-meet, jitsi-videobridge2 and Jibri on-board. Each machine has Ubuntu 22.04 LTS and 8 CPU, 8GB RAM. No any non-default settings except of necessary to conect Jitsi and Jibri between each other were made.
And bug appeared again - after recording of the meeting starting - video quality is degrading, faces of the people starting to be blurred. Used Firefox, Chrome and MS edge - the behaviour is the same in all browsers.
![jitsi_quality](https://github.com/jitsi/jitsi-meet/assets/299814/1651f7bb-e7d8-4f65-82ef-90385c4a78bf)
Here is the list of versions, what we use.
jitsi-meet-prosody/stable,now 1.0.7531-1
jitsi-meet-tokens/stable,now 1.0.7531-1
jitsi-meet-web-config/stable,now 1.0.7531-1
jitsi-meet-web/stable,now 1.0.7531-1
jitsi-meet/stable,now 2.0.8960-1
jitsi-videobridge2/stable,now 2.3-44-g8983b11f-1
lua-basexx/stable,now 0.4.1-jitsi1
lua-cjson/stable,now 2.1.0.10-jitsi1

**Comment by saghul** [comment]
Do you have p2p enabled? What happens if you disable it and have just 2 participants?

**Comment by SignFinder** [comment]
> Do you have p2p enabled? What happens if you disable it and have just 2 participants?

Hi [saghul](https://github.com/saghul) thanks for your questions. We did not change any settings from default.
Now I have changed p2p: enabled: to false and also p2pTestMode: to false and it is magic - we can not reproduce any problems with degrading of video quality. But if I am not wrong - p2p is enabled by default ;-(

**Comment by saghul** [comment]
That is weird, however :-) 

When you enable recording a 3rd hidden participant joins the room to do the recording, so the call would be moved to the JVB. Disabling P2P just makes the call start on the JVB and stay there all the time.

Now, there is a possibility that there is a bug in the "ramp up", aka gauging the right bandwidth estimation when the connection starts, when the connection is switched from P2P to JVB. By disabling P2P you will have already ramped up before the recorder joins.

Can you test a few times to verify that's what happens? With P2P enabled, does the quality eventually go up, if you wait a bit?

**Comment by SignFinder** [comment]
@saghul I have made the following test - p2p is enabled and p2ptestmode = false, two participants.
Before recording started - both participants have good video quality and 720 resolution (for a long time).  After recording is started - video quality was degraded and after some time stream from both participants downgraded to 640x360 (through 990x557 etc.) and it was not restored after 10 minutes of recording.
This behaviour is reproduced several times and in two different test labs - cloud (what placed close to production) and on-premise. 
I also checked load of VMs in labs - all looks fine - a lot of CPU, RAM is free, no any high load avegare or I/O queue.
![jitsi_quality2](https://github.com/jitsi/jitsi-meet/assets/299814/52a67bcb-ed02-4eea-bf89-1f33a466bc2e)
![jitsi_quality3](https://github.com/jitsi/jitsi-meet/assets/299814/d122ca5d-84f7-461a-862a-4b78041b357d)
